docs/tutorial.md

```diff
@@ -368,20 +368,30 @@ _Output_:
 
     replacement for gate CNOT does not preserve the quantum state
 
+##### _CNOT to CZ decomposer_
+
+The decomposition of the `CNOT` gate into a `CZ` gate (with additional single-qubit gates) is used frequently.
+To this end a `CNOT2CZDecomposer` has been implemented that decomposes any `CNOT`s in a circuit to a
+`Ry(-π/2)`-`CZ`-`Ry(π/2)`; the decomposition is illustrated in the image below.
```

Comment:
Suggestion: `. The decomposition`.

Very personal. I don't really like semicolons much haha